src/home/celebration/NftReward.tsx

```diff
@@ -0,0 +1,166 @@
+import { BottomSheetView } from '@gorhom/bottom-sheet'
+import { isPast, isToday } from 'date-fns'
+import differenceInDays from 'date-fns/differenceInDays'
+import React, { useEffect, useMemo, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { openDeepLink } from 'src/app/actions'
+import { BottomSheetRefType } from 'src/components/BottomSheet'
+import BottomSheetBase from 'src/components/BottomSheetBase'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { nftRewardDisplayed } from 'src/home/actions'
+import { isSameNftContract } from 'src/home/celebration/utils'
+import { nftCelebrationSelector, showNftRewardSelector } from 'src/home/selectors'
+import i18n from 'src/i18n'
+import { nftsWithMetadataSelector } from 'src/nfts/selectors'
+import { useDispatch, useSelector } from 'src/redux/hooks'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { formatDistanceToNow } from 'src/utils/time'
+
+export default function NftRewardBottomSheet() {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+
+  const canShowNftReward = useSelector(showNftRewardSelector)
+  const nftCelebration = useSelector(nftCelebrationSelector)
+
+  const nfts = useSelector(nftsWithMetadataSelector)
+  const matchingNft = useMemo(
+    () => nfts.find((nft) => isSameNftContract(nft, nftCelebration)),
+    [nftCelebration]
+  )
+
+  const isVisible = canShowNftReward && matchingNft
+
+  const insets = useSafeAreaInsets()
+  const insetsStyle = { paddingBottom: Math.max(insets.bottom, Spacing.Regular16) }
+
+  const bottomSheetRef = useRef<BottomSheetRefType>(null)
+
+  const expirationDate = new Date(nftCelebration?.expirationDate ?? 0)
+  const rewardReminderDate = new Date(nftCelebration?.rewardReminderDate ?? 0)
+
+  const aboutToExpire = isToday(rewardReminderDate) || isPast(rewardReminderDate)
```

Comment:
I'm sorry, but I'm not getting the idea against `isPast`.

we want to show this bottom sheet in a "reminder state" if the current date falls within the reminder period.
it is not necessary that the user will open the app on the first day of this period, that's why we want to check if the reminder date is in the past.

but I agree that instead of looking at the reminder date we can rely on the `NftCelebrationStatus` instead. thanks for the suggestion!